src/components/ActionBar/ActionBar.test.cy.tsx

```diff
@@ -35,125 +35,160 @@ const nonExpandableActionBar = (props: Partial<ActionBarProps> = {}) => (
+        sizes
+          .filter((s) => s !== size)
+          .forEach((otherSize) => {
+            cy.get('button').parent().invoke('attr', 'class').should('not.include', otherSize);
+          });
+      });
+    });
+    it('should render an actionbar with a classname corresponding to the given color prop', () => {
+      colors.forEach((color) => {
+        mount(expandableActionBar({ color, onClick: cy.stub() }));
+        cy.get('button').parent().invoke('attr', 'class').should('include', color);
+        colors
+          .filter((c) => c !== color)
+          .forEach((otherColor) => {
+            cy.get('button').parent().invoke('attr', 'class').should('not.include', otherColor);
+          });
+      });
+    });
   });
-
-  it('should have aria-expanded=true and not show content when expandable and open=true', () => {
-    cy.mount(expandableActionBar({ open: true, onClick: cy.stub }));
-    cy.get('div').should('contain', 'Content');
-    cy.get('[data-testid="action-bar"]').should('have.attr', 'aria-expanded', 'true');
+  describe('Expandability and actions', () => {
+    it('Should render correctly when all props are set and it is expandable', () => {
+      const actionButton = <Button>Action</Button>;
+      cy.mount(expandableActionBar({ actions: actionButton, open: true, onClick: cy.stub }));
+      cy.get('[data-testid="action-bar-icon"]').should('exist');
+      cy.get('button').should('contain', 'Title');
+      cy.get('button').should('contain', 'Subtitle');
+      cy.get('div').should('contain', 'AdditionalText');
+      cy.get('button').should('contain', 'Action');
+      cy.get('div').should('contain', 'Content');
+    });
+
+    it('should render correctly when all props are set and it is not expandable', () => {
+      const actionButton = <Button>Action</Button>;
+      cy.mount(nonExpandableActionBar({ actions: actionButton }));
+      cy.get('[data-testid="action-bar-icon"]').should('not.exist');
+      cy.contains('Title');
+      cy.contains('Subtitle');
+      cy.contains('AdditionalText');
+      cy.get('button').should('contain', 'Action');
+      cy.get('div').should('not.contain', 'Content');
+    });
+
+    it('non expandable version should not render chevron when it is rendered ', () => {
+      cy.mount(nonExpandableActionBar());
+      cy.get('[data-testid="action-bar-icon"]').should('not.exist');
+    });
+
+    it('should not render chevron when non expandable versjon is rendered ', () => {
+      cy.mount(nonExpandableActionBar());
+      cy.get('[data-testid="action-bar-icon"]').should('not.exist');
+    });
+
+    it('should display content when expandable ActionBar is clicked by mouse (uncontroled open)', () => {
+      cy.mount(expandableActionBar());
+
+      cy.get('[data-testid="action-bar"]').click();
+      cy.contains('Content').should('exist');
+    });
+
+    it('should call handleClick when one is provided and expandable ActionBar is clicked by mouse (controled open)', () => {
```

Comment:
Change to: "should call handleClick when it is provided and expandable ActionBar is clicked by mouse (controled open)"